index.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="author" content="Ramiro, Standley">
+    <meta name="description" content="Trial Microverse">
+    <meta name="keywords" content="DOM, BOM, javascript, html">
+    <link rel="stylesheet" href="../css/bootstrap.min.css">
+    <link rel="stylesheet" href="../css/style.css">
+    <script src="https://kit.fontawesome.com/35e1f586d0.js" crossorigin="anonymous"></script>
+    <title>Tribute page</title>
+</head>
+<body>
+  <header id="header">
+    <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
+        <div class="container">
+          <a class="navbar-brand titulo-cursiva active" aria-current="page" href="../index.html">
+            <p class="textIni"> <img id="header-image" src="../img/logo.jpeg" width="100" height="100" class="logoNav" alt="landing logo"> Albert Einstein
```

Comment:
- [ ] Your comment should have an explanation of Why you requested the changes, kindly add this

Reply:
@Ismailco Thank you very much for your review. However, I don't understand what you are requesting here: "This PR should be invalidated because it has no description, kindly pay attention to this". please can you elaborate a little more?  I'm fixing the second issue now. Thank you once more for your time

Reply:
@StarMindz, sorry for the delay
I mean that the PR has no description, therefore it should be invalidated

- This is the PR comment:
![Screenshot_20220704_205207](https://user-images.githubusercontent.com/25227022/177210877-2e5078e0-ff29-4a14-a228-01bb6e599726.png)

Reply:
@Ismailco, OH, I understand now, I've made all the needed changes. Thank you very much for your review